contracts/interfaces/IGnosisAuction.sol

```diff
@@ -68,4 +68,9 @@ interface IGnosisAuction {
         bytes32[] memory _prevSellOrders,
         bytes calldata allowListCallData
     ) external returns (uint64);
+
+    function claimFromParticipantOrder(
```

Comment:
bidder actually needs to call [claimFromParticipantOrder](https://github.com/gnosis/ido-contracts/blob/a766a8b30de14781961158b772393cc49b900714/contracts/EasyAuction.sol#L568) as well. so will need to call this on behalf of delta vault in cron job in separate pr

Reply:
Do we want to make a pass-through function on the Delta Vault that calls `claimFromParticipantOrder`? Feels better to have everything on 1 contract so people can poke it